dora/core/common/src/test/java/alluxio/UfsUrlTest.java

```diff
@@ -35,21 +42,23 @@ public class UfsUrlTest {
   /**
    * Tests the {@link UfsUrl#createInstance(String)} constructor for basic Alluxio paths.
    */
+
   @Test
   public void basicUfsUrl() {
     UfsUrl ufsUrl = UfsUrl.createInstance("abc://localhost:19998/xy z/a b c");
 
-    assertTrue(ufsUrl.getAuthority().isPresent());
-    assertEquals("localhost:19998", ufsUrl.getAuthority().get().toString());
-    SingleMasterAuthority authority = (SingleMasterAuthority) ufsUrl.getAuthority().get();
+    assertNotNull(ufsUrl.getAuthority());
+    assertEquals("localhost:19998", ufsUrl.getAuthority().toString());
+    SingleMasterAuthority authority = (SingleMasterAuthority) ufsUrl.getAuthority();
```

Comment:
The existing Authority class implementations are more describing the formats of authority in "an alluxio url", like EmbeddedLogicalAuthority and ZookeeperLogicalAuthority etc.

As we are writing a UfsUrl, the authority names should no longer be in those formats. We should be looking at the authority formats for a UFS, like HDFS vs S3(maybe empty). We may need to throw away the old implementations like `EmbeddedLogicalAuthority` etc.

No need to change anything in this PR. Let's migrate one step at a time.